pyproject.toml

```
@@ -0,0 +1,71 @@
+[project]
+name = "log-archival-bench"
+version = "0.0.1"
```

```suggestion
version = "0.1.0"
```

Semver recommends starting at 0.1.0.